[2024-12-12T08:27:46.333+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-12T08:27:46.355+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_orchestrator.test_connection.dbt_test_connection scheduled__2024-12-12T08:00:00+00:00 [queued]>
[2024-12-12T08:27:46.362+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_orchestrator.test_connection.dbt_test_connection scheduled__2024-12-12T08:00:00+00:00 [queued]>
[2024-12-12T08:27:46.362+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-12-12T08:27:46.374+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): test_connection.dbt_test_connection> on 2024-12-12 08:00:00+00:00
[2024-12-12T08:27:46.380+0000] {standard_task_runner.py:63} INFO - Started process 831 to run task
[2024-12-12T08:27:46.382+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt_orchestrator', 'test_connection.dbt_test_connection', 'scheduled__2024-12-12T08:00:00+00:00', '--job-id', '219', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpg140a37w']
[2024-12-12T08:27:46.383+0000] {standard_task_runner.py:91} INFO - Job 219: Subtask test_connection.dbt_test_connection
[2024-12-12T08:27:46.426+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt_orchestrator.test_connection.dbt_test_connection scheduled__2024-12-12T08:00:00+00:00 [running]> on host 34fdc836a0bb
[2024-12-12T08:27:46.510+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ma7moud' AIRFLOW_CTX_DAG_ID='dbt_orchestrator' AIRFLOW_CTX_TASK_ID='test_connection.dbt_test_connection' AIRFLOW_CTX_EXECUTION_DATE='2024-12-12T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-12T08:00:00+00:00'
[2024-12-12T08:27:46.511+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-12T08:27:46.523+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-12-12T08:27:46.523+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'dbt debug --profiles-dir /opt/***/dbt --project-dir /opt/***/dbt/ecommerce']
[2024-12-12T08:27:46.536+0000] {subprocess.py:86} INFO - Output:
[2024-12-12T08:27:48.343+0000] {subprocess.py:93} INFO - [0m08:27:48  Running with dbt=1.8.0
[2024-12-12T08:27:48.353+0000] {subprocess.py:93} INFO - [0m08:27:48  dbt version: 1.8.0
[2024-12-12T08:27:48.354+0000] {subprocess.py:93} INFO - [0m08:27:48  python version: 3.9.19
[2024-12-12T08:27:48.354+0000] {subprocess.py:93} INFO - [0m08:27:48  python path: /home/***/.local/bin/python
[2024-12-12T08:27:48.355+0000] {subprocess.py:93} INFO - [0m08:27:48  os info: Linux-6.8.0-49-generic-x86_64-with-glibc2.36
[2024-12-12T08:27:48.420+0000] {subprocess.py:93} INFO - [0m08:27:48  Using profiles dir at /opt/***/dbt
[2024-12-12T08:27:48.420+0000] {subprocess.py:93} INFO - [0m08:27:48  Using profiles.yml file at /opt/***/dbt/profiles.yml
[2024-12-12T08:27:48.421+0000] {subprocess.py:93} INFO - [0m08:27:48  Using dbt_project.yml file at /opt/***/dbt/ecommerce/dbt_project.yml
[2024-12-12T08:27:48.422+0000] {subprocess.py:93} INFO - [0m08:27:48  adapter type: postgres
[2024-12-12T08:27:48.423+0000] {subprocess.py:93} INFO - [0m08:27:48  adapter version: 1.8.0
[2024-12-12T08:27:48.497+0000] {subprocess.py:93} INFO - [0m08:27:48  [[33mWARNING[0m]: Deprecated functionality
[2024-12-12T08:27:48.497+0000] {subprocess.py:93} INFO - The `tests` config has been renamed to `data_tests`. Please see
[2024-12-12T08:27:48.497+0000] {subprocess.py:93} INFO - https://docs.getdbt.com/docs/build/data-tests#new-data_tests-syntax for more
[2024-12-12T08:27:48.497+0000] {subprocess.py:93} INFO - information.
[2024-12-12T08:27:48.541+0000] {subprocess.py:93} INFO - [0m08:27:48  Configuration:
[2024-12-12T08:27:48.542+0000] {subprocess.py:93} INFO - [0m08:27:48    profiles.yml file [[32mOK found and valid[0m]
[2024-12-12T08:27:48.542+0000] {subprocess.py:93} INFO - [0m08:27:48    dbt_project.yml file [[32mOK found and valid[0m]
[2024-12-12T08:27:48.543+0000] {subprocess.py:93} INFO - [0m08:27:48  Required dependencies:
[2024-12-12T08:27:48.552+0000] {subprocess.py:93} INFO - [0m08:27:48   - git [[32mOK found[0m]
[2024-12-12T08:27:48.553+0000] {subprocess.py:93} INFO - 
[2024-12-12T08:27:48.553+0000] {subprocess.py:93} INFO - [0m08:27:48  Connection:
[2024-12-12T08:27:48.554+0000] {subprocess.py:93} INFO - [0m08:27:48    host: 172.18.0.3
[2024-12-12T08:27:48.554+0000] {subprocess.py:93} INFO - [0m08:27:48    port: 5432
[2024-12-12T08:27:48.555+0000] {subprocess.py:93} INFO - [0m08:27:48    user: ***
[2024-12-12T08:27:48.555+0000] {subprocess.py:93} INFO - [0m08:27:48    database: ecommerce
[2024-12-12T08:27:48.556+0000] {subprocess.py:93} INFO - [0m08:27:48    schema: oltp
[2024-12-12T08:27:48.556+0000] {subprocess.py:93} INFO - [0m08:27:48    connect_timeout: 10
[2024-12-12T08:27:48.557+0000] {subprocess.py:93} INFO - [0m08:27:48    role: None
[2024-12-12T08:27:48.557+0000] {subprocess.py:93} INFO - [0m08:27:48    search_path: None
[2024-12-12T08:27:48.558+0000] {subprocess.py:93} INFO - [0m08:27:48    keepalives_idle: 0
[2024-12-12T08:27:48.558+0000] {subprocess.py:93} INFO - [0m08:27:48    sslmode: None
[2024-12-12T08:27:48.559+0000] {subprocess.py:93} INFO - [0m08:27:48    sslcert: None
[2024-12-12T08:27:48.559+0000] {subprocess.py:93} INFO - [0m08:27:48    sslkey: None
[2024-12-12T08:27:48.560+0000] {subprocess.py:93} INFO - [0m08:27:48    sslrootcert: None
[2024-12-12T08:27:48.560+0000] {subprocess.py:93} INFO - [0m08:27:48    application_name: dbt
[2024-12-12T08:27:48.561+0000] {subprocess.py:93} INFO - [0m08:27:48    retries: 1
[2024-12-12T08:27:48.562+0000] {subprocess.py:93} INFO - [0m08:27:48  Registered adapter: postgres=1.8.0
[2024-12-12T08:27:48.768+0000] {subprocess.py:93} INFO - [0m08:27:48    Connection test: [[32mOK connection ok[0m]
[2024-12-12T08:27:48.768+0000] {subprocess.py:93} INFO - 
[2024-12-12T08:27:48.769+0000] {subprocess.py:93} INFO - [0m08:27:48  [32mAll checks passed![0m
[2024-12-12T08:27:49.743+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-12T08:27:49.744+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-12T08:27:49.767+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dbt_orchestrator, task_id=test_connection.dbt_test_connection, run_id=scheduled__2024-12-12T08:00:00+00:00, execution_date=20241212T080000, start_date=20241212T082746, end_date=20241212T082749
[2024-12-12T08:27:49.803+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-12T08:27:49.842+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-12T08:27:49.844+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
